6_visualize/src/create_gif_tasks.R

```diff
@@ -39,6 +39,23 @@ create_intro_gif_tasks <- function(intro_config, folders, storm_start_date){
       )
     }
   )
+  legend_frame <- scipiper::create_task_step(
```

Comment:
Huh. Haven't I already been working from code that included this? I thought you had added a version where DateTime was dynamic (for the storm animation) and then I had added a version where DateTime was NA (for the intro animation): https://github.com/USGS-VIZLAB/vizstorm-GIF/pull/106/files#diff-0de26640bfa961faaa7a7265e37434e9R43. Maybe this was a merge issue because line 60 just below starts another bunch of lines [re]defining `legend_frame`.

Reply:
weird...perhaps this was related to what was branched from master to create this new branch on `upstream`? Maybe it didn't have a recent PR...which is odd, because I thought I just created this branch on USGS-R.

Reply:
Yeah, and it says it's up to date. Maybe it had to do with overlap between when my PR was open and when yours was?

Reply:
odd, I will clean it up.